src/Xamarin.Android.Build.Tasks/Tasks/GetJavaPlatformJar.cs

```diff
@@ -58,22 +60,39 @@ public override bool RunTask ()
 								platform = target_sdk.Value;
 
 							var min_sdk = uses_sdk.Attribute (androidNs + "minSdkVersion");
-							if (min_sdk != null && (!int.TryParse (min_sdk.Value, out int minSdkVersion) || minSdkVersion < XABuildConfig.NDKMinimumApiAvailable)) {
-								Log.LogWarningForXmlNode (
-										code:             "XA4216",
-										file:             AndroidManifest,
-										node:             min_sdk,
-										message:          Properties.Resources.XA4216_MinSdkVersion,
-										messageArgs:      new object [] { min_sdk?.Value, XABuildConfig.NDKMinimumApiAvailable }
-								);
+							if (min_sdk != null) {
```

Comment:
Aside: the amount of indentation going on here has my crying out for a helper method 3 levels of indentation ago…